doc/api/errors.md

```diff
@@ -1235,8 +1235,7 @@ type for one of its returned object properties on execution.
 ### ERR_INVALID_RETURN_VALUE
 
 Thrown in case a function option does not return an expected value
-type on execution.
-For example when a function is expected to return a promise.
+type on execution. For example, when a function is expected to return a promise.
```

Comment:
I know this did not originate in this PR, but the second sentence is a fragment. Optional nit: Make it a complete sentence, probably by appending it to the previous sentence and changed `For example` to `such as`: `...type on execution, such as when a function is expect to return a promise`.